generic-indexer/run.sh

```
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(cd $(dirname $0) ; pwd -P)
+
+TASK=$1
+ARGS=${@:2}
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)